src/osemgrep/cli_ci/Test_is_blocking_helpers.ml

```diff
@@ -166,7 +167,9 @@ let tests =
               in
               List.iter
                 (fun (name, m, expected) ->
-                  Alcotest.(check bool) name expected (finding_is_blocking m))
+                  Alcotest.(check bool)
+                    name expected
+                    (finding_is_blocking (m ())))
```

Comment:
```suggestion
                    (finding_is_blocking m))
```
would rather this + change the calls to `cli_match_of_finding_with_actions` to be fully applied.

Reply:
done in [15a5734](https://github.com/semgrep/semgrep/pull/9943/commits/15a57340f1e62db6ea57811b7fe86151d4197bdf)